keyboards/giabalanai/keymaps/default_giabarinaix2/rules.mk

```diff
@@ -1,3 +1 @@
 ENCODER_ENABLE = no        # encoder on mute button
-
```

Comment:
encodermap not being used on thes keymaps?

Reply:
No, encodermap is not used in this keymap.
I forgot to disable it in giabarinaix2led keymap. I added it there as well.